acme/acme/__init__.py

```diff
@@ -8,13 +8,14 @@
 import sys
 import warnings
 
+import josepy as jose
```

Comment:
We probably want to undo this, since our convention is usually to have the comment precede the code.

Reply:
https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments

Reply:
Fixed.